homework-submission/react-hyf/client/src/hoc/Layout/layout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import NavBar from '../../components/Navbar';
+
+const Layout = props => {
+  return (
+    <div>
```

Comment:
`Layout` isn't a `HOC`, it's a basically a stateless component.
i would make a `layout` folder inside `src` or `components` and put `Layout.js` there

HOC: https://reactjs.org/docs/higher-order-components.html